terraform/modules/benchmark/service/bench.rb

```diff
@@ -149,15 +149,22 @@ def write_csv(metrics, options)
     timestamp = Time.now.strftime('%Y-%m-%d %H:%M:%S')
     chain = options[:chain]
 
-    results = { import_time: { forest: 'n/a', lotus: 'n/a' },
-                validation_time: { forest: 'n/a', lotus: 'n/a' } }
+    results = {
+      import_time: { forest: 'n/a', lotus: 'n/a' },
+      validation_time: { forest: 'n/a', lotus: 'n/a' },
+      peak_memory: { forest: 'n/a', lotus: 'n/a' }
+    }
 
     metrics.each do |key, value|
       elapsed = value[:import][:elapsed] || 'n/a'
       tpm = value[:validate_online][:tpm] || 'n/a'
+      peak_memory_import = value[:import][:peak_memory] || 'n/a'
+      peak_memory_validate = value[:validate_online][:peak_memory] || 'n/a'
+      peak_memory = [peak_memory_import, peak_memory_validate].max
```

Comment:
This will throw an exception if classes of `peak_memory_import` and `peak_memory_validate` do not match.

```
irb(main):005:0> ['n/a', 3].max
(irb):5:in `<main>': comparison of Integer with String failed (ArgumentError)
```